code/game/objects/items/weapons/manuals.dm

```diff
@@ -113,88 +113,44 @@
+	name = "Supermatter Engine Anomaly Refrence"
+	desc = "An engineer's best tool for dealing with their worst frenemy: The Supermatter and its anomalous behavior."
 	icon_state = "bookParticleAccelerator"
-	author = "Waleed Asad"
-	title = "Supermatter Engine User's Guide"
+	author = "Vroo-Looum-Kloo"
+	title = "Supermatter Engine Anomaly Refrence"
 
 	pages = list({"Engineering notes on single-stage Supermatter engine,</br>
-			-Waleed Asad</br>
+			Vroo-Looum-Kloo</br>
 
-			A word of caution, do not enter the engine room, for any reason, without radiation protection and mesons on. The status of the engine may be unpredictable even when you believe it is .off.. This is an important level of personal protection.</br></br>
+			The supermatter engine is a very powerful, yet strange method of power generation. This guide will serve as a pocket refrence of the myraid of anomalous behaviors it may exhibit thoughout your shift. Below is a table of events and their effect.</br></br></br>
 
-			The engine has two basic modes of functionality. He has observed that it is capable of both a safe level of operation and a modified, high output mode.</br></br>
+			D Class: Events that only effect certain types of NON-STANDARD setups, minimial operator intervention required, they occur instantly and engineering will be alerted on telecomms.</br></br>
 
-			<center><b>Notes on starting the basic function mode, dubbed .Heat-Primary Mode..</b></center></br></br>
+			D-1: 200 mols of nitrous oxide are released by the crystal.</br>
+			D-2: 200 mols of nitrogen are released by the crystal</br>
+			D-3: 250 mols of CO2 are released by the crystal</br></br></br>
 
-			1. Prepare collector arrays. This is done standard to any text on their function by wrenching them down, filling six plasma tanks with a plasma canister, and inserting the tank into the collectors one by one. Finally, initialize each collector.</br></br>
+			C Class: Events with mild effects to standard setups. Operator intervetion MAY be required. Engineering will be alerted on telecomms.</br></br>
+			C-1: 250 mols of oxygen are released by the crystal</br>
+			C-2: 250 mols of plasma are released by the crystal</br>
+			C-3: The temperature at which the SM starts to lose integrity is lowered for a few minutes.</br></br></br>
 
-			2. Prepare gas system. Before introducing any gas to the Supermatter engine room, it is important to remember the small but vital steps to preparing this section. First, set the input gas pump and output gas flow pump to 4500, or maximum flow. Second, switch the digital switching valve into the .up. position, in order to circulate the gas back toward the coolers and collectors.</br></br>
+			B Class: Events with significant effects to standard setups. SCRAM protocol may be required.</br></br>
+			B-1: The amount of plasma and O2 released by the SM is doubled for a few minutes.</br>
+			B-2: The amount of heat released by the SM is multiplied by 1.25 for a few minutes.</br>
```

Comment:
I think these might be some pretty specific numbers for an in-game reference manual. I'd suggest being a bit less specific to keep some of the magic numbers hidden away.

Reply:
Will do!